src/org/opensolaris/opengrok/configuration/RuntimeEnvironment.java

```diff
@@ -1552,6 +1561,11 @@ public void setConfiguration(Configuration configuration, List<String> subFileLi
         // The invalidation of repositories above might have excluded some
         // repositories in HistoryGuru so the configuration needs to reflect that.
         configuration.setRepositories(new ArrayList<>(histGuru.getRepositories()));
+        
+        configuration.getBodyIncludeFileContent(true);
```

Comment:
I, personally, would get rid of the `force` param and would add method like `resetIncludeFiles` or something similar. I think the intent would be more clear.

Reply:
Somehow I thought the force flag was quite elegant, not having to duplicate the calls to `getFileContent()`. What I could do is to move these 4 calls from `setConfiguration()` into a private method.